### PR TITLE
Automate WSO2 API Manager deploy with Ansible

### PR DESCRIPTION
**Pull request by senthuja**
## Purpose
Without download the wso2am-2.2.0.zip and unzip it from the wso2 website and manage more than one at a time using ansible.

## Goals
Write the playbook for automate wso2 api manager (version-2.2.0) deployment with ansible.

## Approach
Reffer this [link]([url](https://docs.ansible.com/ansible/2.5/user_guide/playbooks_best_practices.html))

## User stories
Automate WSO2 API Manager deployment with Ansible

Background
The user’s machine connect with openssh and start the wso2am-2.2.0 without executing command.

User
WSO2 API Manager User

Prerequisites
Connect with OPENSSH (no need of password authentication).
All WSO2 Carbon-based products are Java applications that can be run on any platform that is Oracle JDK 1.7/1.8 .
Start WSO2 API Manage without  executing wso2server.sh at  /<home>/bin using ansible playbook.
Unzip the downloaded MySQL driver archive, and copy the MySQL JDBC driver JAR (mysql-connector-java--5.1.30-bin.jar) into the <home>/repository/components/lib directory in all the nodes in the cluster.

Description
Install Oracle Java SE Development Kit (JDK) version 1.7.* or 1.8.* and set the JAVA_HOME environment variable using playbook.
Download WSO2 API Manager and Start WSO2 API Manager and without executing wso2server.sh (for Linux) using playbook.
The configuration of wso2am-2.2.0 may change in the vars directory.

## Release note
Deploy the apim to many customers with a certain profile.

## Documentation
https://docs.google.com/document/d/1w66-NoLfVqRRhrywbWJpTirqaCeJ_GVnsYZ7sRjnr-Y/edit?usp=sharing

## Training
N/A

## Certification
N/A

## Marketing
https://blinkingstaruoj.blogspot.com/

## Automation tests
 N/A

## Security checks
 N/A

## Samples
N/A

## Related PRs
N/A

## Migrations (if applicable)
N/A

## Test environment
Supporting Java version
jdk1.8.0_131
Supporting Operating System 
Ubuntu 16.04
Supporting Ansible Version
Ansible 2.4.3.0

## Learning
In conclusion, at this stage of Ansible use, I would say it is a great application to use and would recommend it to others.

### CODE REVIEW

**Comment by chamilad** [comment]
@KavinduZoysa Please approve at the current state for merging.

**Comment by senthuja** [comment]
Hi all,
Thanks.

Regards,
K.Senthuja
